.github/workflows/kb_sdk_test.yaml

```diff
@@ -56,9 +43,12 @@ jobs:
     - name: Run tests
       if: "!contains(github.event.head_commit.message, 'skip ci')"
       shell: bash
-      env:
-        CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
       run: |
-        sh $GITHUB_WORKSPACE/kb_sdk_actions/bin/kb-sdk test
-        bash <(curl -s https://codecov.io/bash)
+        sh scripts/GHA/kb-sdk test --verbose
+
+    # - name: Upload coverage to Codecov
+    #   uses: codecov/codecov-action@v3
+    #   with:
+    #     token: ${{ secrets.CODECOV_TOKEN }}
+    #     fail_ci_if_error: true
```

Comment:
it looks like coverage is set up for this module:

https://github.com/kbase/genome_annotation_api/blob/master/Makefile#L46-L58

Reply:
My last commit did not comment codecov and it failed to find coverage file though.

Reply:
Based on the makefile coverage is running; you'll need to figure out where the coverage file is ending up and how to get codecov to find it

Reply:
fixed